src/DotLiquid/Tags/AddFilters.cs

```diff
@@ -16,7 +16,7 @@ public class AddFilters : Tag
     {
         private static readonly Regex Syntax = R.B(R.Q(@"\s*([""'].+[""'])\s*"));
 
-        private static readonly IDictionary<string, Type> Whitelist = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+        private static readonly IDictionary<string, Type> _Whitelist = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Why the extra '_'? I think I saw that on some of the code in this repo, but that's rather old school arcane flagging of variable names. And then, why is the also static readonly Syntax about not getting an underscore prefix?

If we want clarity when using the variables whether those are instance variable or static, we should prefix the call with `this.` or with the class name `AddFilters.`.

Reply:
The static dictionary Whitelist clashes with the method Whitelist.
I agree the `_` is old school, so I've made a rename.